tests/macros/utils.js

```diff
@@ -176,17 +176,22 @@ function lintHTML(html, fragment = true) {
                 </html>`;
     }
     try {
-        execSync(`java -jar ${vnu} --errors-only --format text -`, {
+        execSync(`java -jar ${JSON.stringify(vnu)} --errors-only --format text -`, {
             input: html,
             stdio: 'pipe',
             timeout: 15000
         });
         return null;
     } catch (error) {
         const error_message = error.message
-            .split(os.EOL)
-            .filter(line => line.startsWith('Error: '))
+            // `vnu` always uses `\n`, even on Windows.
+            .split(/\r?\n/g)
```

Comment:
If it's always `\n` (even on Windows) then why split on possibly `\r`?

Reply:
In a future case if `vnu` ever gets updated and starts adding `\r` on Windows.